test/extended/cluster/utils.go

```diff
@@ -519,3 +519,37 @@ func SetNamespaceLabels(c kclientset.Interface, name string, labels map[string]s
 	ns.Labels = labels
 	return c.CoreV1().Namespaces().Update(ns)
 }
+
+func ProjectExists(oc *exutil.CLI, name string) (bool, error) {
+	p, err := oc.AdminProjectClient().Project().Projects().Get(name, metav1.GetOptions{})
+	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
```

Comment:
Checking for error text isn't usually a good pattern, we want to filter based on error type instead.

Reply:
Good point. Let me try the type way, although it is rom [WaitUntilPodIsGone](https://github.com/openshift/origin/blob/master/test/extended/util/framework.go#L1056).

Reply:
That's unfortunate, thanks @hongkailiu